network_tests/test_load_ckpts.py

```diff
@@ -36,3 +36,36 @@ def test_load_bert_base(self, weights):
             "padding_mask": tf.constant([1] * 512, shape=(1, 512)),
         }
         model(input_data)
+
+    def test_load_bert_tiny(self, weights):
```

Comment:
Could we just make one big parameterized test? Might actually improve the readability overall, and save some lines.

    @parameterized.named_parameters(
        ("base_uncased_en", keras_nlp.models.BertBase, "uncased_en"),
    )
    def test_load_bert_base(self, model, weights):